### PR TITLE
Dokka fixes

### PR DESCRIPTION
**Pull request by sschuberth**
Please have a look at the individual commit messages for the details.

### CODE REVIEW

File: evaluator/src/main/kotlin/Rule.kt [hunk 1]
```diff
@@ -105,6 +105,11 @@ abstract class Rule(
      */
     abstract fun issueSource(): String
 
+    /**
+     * Add an issue of the given [severity] for [pkgId] to the list of violations. Optionally, the offending [license]
+     * and [source][licenseSource] of the license can be specified. The [message] further explains the violation itself
```
Comment (fviernau) [inline evaluator/src/main/kotlin/Rule.kt]: Did you mean `[licenseSource]` instead of `[source][licenseSource]`?

Comment (sschuberth) [reply evaluator/src/main/kotlin/Rule.kt]: Actually, no. Just "source" reads nicer as part of the text, but I still want to link that text to `licenseSource`.

Comment (fviernau) [reply evaluator/src/main/kotlin/Rule.kt]: ok, thanks!

Comment (mnonnenmacher) [reply evaluator/src/main/kotlin/Rule.kt]: Suggestion: `...the offending [license] and its [source][licenseSource] can be specified...`

Comment (sschuberth) [reply evaluator/src/main/kotlin/Rule.kt]: Done.

**Pull request merged**